packages/fcl/src/wallet-provider-spec/draft-v3.md

```diff
@@ -0,0 +1,548 @@
+## Status
+
+- **Last Updated:** August 3rd 2021
+- **Stable:** Yes
+- **Risk of Breaking Change:** Medium
+- **Compatibility:** `>= @onflow/fcl@0.0.77`
+
+# Overview
+
+Flow Client Library (FCL) approaches the idea of blockchain wallets on Flow in a different way than how wallets may be supported on other blockchains. For example, with FCL, a wallet is not necessarily restricted to being a browser extention or even a native application on a users device. Wallets on Flow can take the shape of being many different types applications. Since wallet applications can take on many forms, we needed to create a way for these varying applications to be able to communicate and work together.
+
+FCL acts in many ways as a protocol to facilitate communication and configuration between the different parties involved in a blockchain application. An _Application_ can use FCL to _authenticate_ users, and request _authorizations_ for transactions, as well as mutate and query the _Blockchain_. An application using FCL offers it's _Users_ a way to connect and select any number of Wallets Providers and their Wallet Services. A selected _Wallet_ provides an Applications instance of FCL with configuration information about itself and its Wallet Services, of which the _User_ and _Application_ can interact with.
```

Comment:
A authorized _Wallet_ provides the application's instance of FCL with configuration information about itself and its services, allowing the _User_ and _Application_ to interact with them.

Reply:
I like this, I just want to avoid the use of the word _authorized_ since its a bit of a loaded term in our context